django_future_tasks/management/commands/process_future_tasks.py

```diff
@@ -29,6 +29,9 @@ def add_arguments(self, parser):
         )
 
     def _handle_termination(self, *args, **kwargs):
+        for task in self.tasks_in_progress():
```

Comment:
I would sugest to extend the `tasks_for_processing` in order to store the list of task or store the task that is currently active in order to be makred as intrerupted. This would allow us to run multiple workers in future and cancel them separately.

Reply:
Yes, makes sense. I've changed that now.